src/smrt/shift_nek_driver.cpp

```diff
@@ -93,7 +93,25 @@ void ShiftNekDriver::solve()
 {
   // Loop to convergence or fixed iteration count
   for (int iteration = 0; iteration < max_picard_iter_; ++iteration) {
-    // Set heat source in Nek
+    // Set heat source in Nek; first, get heat source indexed by cell ID
+    // from Shift and map it to Nek elements
+    for (int cellid = 0; cellid < d_power_map.size(); ++cellid) {
+      const auto& elem_list = d_power_map[cellid];
```

Comment:
The usage of `d_power_map` here was migrated from `ShiftDriver` but its declaration and initialization were not moved.

Reply:
You mean that `d_power_map` is initialized in `ShiftDriver` but only used in `ShiftNekDriver`, so it should logically be moved to `ShiftNekDriver` instead?

If so, I was planning on doing this in a future MR - didn't want to do too much without being able to compile.

Reply:
For the time being, it doesn't matter to me if `d_power_map` sits in `ShiftDriver` or in `ShiftNekDriver`.  The problem is that it is declared as a member variable (and initialized in `set_centroids_and_volumes`) in `ShiftDriver` but it is used here in `ShiftNekDriver` so the code won't compile as-is.  It seems like it got half-migrated.

Reply:
Oh, good catch!